lib/govuk_schemas/random_helpers.rb

```diff
@@ -0,0 +1,126 @@
+module GovukSchemas
+  # @private
+  class RandomHelpers
```

Comment:
Rather than naming this RandomHelpers, which makes it sound a bit like a bag of functions, I'd consider calling it RandomContentGenerator.